middleware/common/source/service/call/context.cpp

```diff
@@ -251,12 +254,11 @@ namespace casual
 
 
          reply::Result result;
-         auto prepared = get_reply();
+         auto [ reply, desc] = get_reply();
```

Comment:
Perhaps **decriptor** should be used iso **desc** which can be mistaken for _descending_.